addons/intercom_phone/functions/fnc_getIntercomChannel.sqf

```diff
@@ -0,0 +1,28 @@
+#include "script_component.hpp"
+
+/*
+  Name: TFAR_fnc_getIntercomChannel
+
+  Author: Arend(Saborknight)
+    Gets the intercom channel in use by the crew of a vehicle,
+    in the order of: Driver, Gunner, Commander, Turrets, Cargo
+
+  Arguments:
+    0: Vehicle object <OBJECT>
+
+  Return Value:
+    Intercom Channel <NUMBER>
+
+  Example:
+    [_vehicle, _player] call TFAR_fnc_getIntercomChannel;
+
+  Public: Yes
+*/
+
+params ["_vehicle"];
+
+// Get any occupants, from crew down to passengers
+private _crew = crew _vehicle;
```

Comment:
It actually sucks that `crew` actually returns the list sorted by driver, gunner, commander and then passengers... when in actual fact it makes more sense to connect to the commander's channel, then the gunner's, then the driver's, then the passengers if the previous are not present...